docs/src/dataloader.md

```diff
@@ -168,3 +188,50 @@ public interface IUsersStore
 	Task<IEnumerable<User>> GetAllUsersAsync();
 }
 ```
+## Usage *without* Dependency Injection
+Because the DataLoaderContext is defined in the GraphQL Root object we can get it in our resolvers like this:
+```csharp
+public class ordertype : ObjectGraphType<order>
+{
+    public ordertype()
+    {
+        Name = "Ordertype";
+        UserStore store = new UserStore();
+
+        Field<IntGraphType>("id");
+        Field<StringGraphType>("userId");
+        Field<IntGraphType>("category");
+        Field<StringGraphType>("name");
+        Field<DateGraphType>("date");
+        Field<Usertype, User>.Name("user").ResolveAsync(ctx =>
+        {
+            store.SetResolveContext(ctx); //Make ctx availabele
+            var loader = ctx.RootValue.Context as DataLoaderContext.GetOrAddBatchLoader<int, User>("GetUserIds", store.GetUserIds);
+
+            return loader.LoadAsync(ctx.Source.userId);
+        });
+    }
+}
+
+```
+And we can define the UserStore like this:
+```csharp
+public class UserStore
```

Comment:
The sample above uses a IUserStore interface.  I think it would help the reader to keep with that interface definition vs create an entirely new sample with basically the same name.  For example, show the class implementing that interface to keep continuity. I think the implementation is somewhat irrelevant to the sample.

Reply:
Yes I agree that it creates inconcistency. But a question... did I just miss something or how can something happen in the interface? The interface is basically only a skeleton of what needs to be implemented? But there is no function?